src/com/pentalog/pentastagiu/homework2/Product.java

```diff
@@ -0,0 +1,29 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class Product {
+    private int id;
+    private String name;
+    private double price;
+
+    public Product(int id, String name, double price) {
+        this.id = id;
```

Comment:
id-ul ar fi trebuit tratat altfel, de vreme ce reprezinta un identificator pentru obiectul nostru, pe care clientul nu ar trebui sa il paseze in constructor: o sugestie de implementare e folosirea unui field static counter, care ne numara cate instante dintr-un obiect au fost create, iar in constructor id-ul nu va fi trimis ca parametru ci va fi setat de noi cu valoarea counterului

totodata in constructor si metode publice(mai ales setteri sau alte metode care modifica starea obiectului), trebuie sa ne asiguram ca avem toate validarile necesarea pentru a impiedica crearea unui obiect invalid(cu pret sau cantitate negativa)

pentru implementari in general ti-as sugera sa incepi cu fielduri private iar getteri/setteri sa adaugi pe masura ce va fi nevoie